<?xml version="1.0"?>
<launch>

    <arg name="id" default="1" />
    <arg name="pos_type" default="local" />
    <arg name="x" default="0" />
    <arg name="y" default="0" />
    <arg name="output" default="log" />

    <!-- Position provider -->
    <node pkg="mavros_pos_provider" type="mavros_pos_provider" name="mavros_pos_provider_$(arg id)" output="$(arg output)" required="true">
        <param name="pos_type" value="$(arg pos_type)" />
        <param name="x" value="$(arg x)" />
        <param name="y" value="$(arg y)" />
        <rosparam file="$(find mavros_pos_provider)/param/mavros_pos_provider.yaml" command="load" />
    </node>

</launch>
